docs/knowledge-base/ts-server-boilerplate/repo.md

```diff
@@ -0,0 +1,140 @@
+---
+id: repo
+title: Repo configuration and validation
+sidebar_position: 3
+---
+In order to make the boilerplate tick, we have many configuration files and scripts that help us maintain the codebase. This document will guide you through all of them.
+
+## Dependabot
+Dependabot is a GitHub app that checks for outdated dependencies in your repository and creates pull requests to update them. The configuration file for Dependabot is located in the `.github` directory.
+If you need to change the configuration, you can do so by editing the `.github/dependabot.yml` file.
+For reference on how to configure Dependabot, check the [official documentation](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference).
+
+## workflows
+The workflows for GitHub Actions are located in the `.github/workflows` directory. The boilerplate includes the following workflows:
+ - `build-and-push.yaml`: This workflow is triggered when a new version is released. It builds the Docker image and pushes it to the Docker registry.
+ - `pull_request.yaml`: This workflow is triggered when a pull request is opened. It runs the tests and checks the code style.
+ - `release-please.yaml` This workflow manages the release of the package using [release-please](https://github.com/googleapis/release-please).
+
+For more information check the [Pipelines docs](../DevOps/pipelines/)
```

Comment:
The `pipelines` link is broken.

Reply:
fixed

Reply:
It still seems broken for some reason (at least in the preview).